FX/frabert_ogler.ext

```diff
@@ -0,0 +1,22 @@
+@description Ogler
```

Comment:
I think this could use a more descriptive display name. How about something similar to the forum thread's title?

```suggestion
@description Ogler: FX plugin for using OpenGL shaders in REAPER
```